ArduCopter/control_auto.cpp

```diff
@@ -168,10 +168,10 @@ void Copter::auto_takeoff_run()
     }
 
     // process pilot's yaw input
-    float target_yaw_rate = 0;
+    float target_yaw_rate_cdps = 0; // centi-degrees / sec
     if (!failsafe.radio) {
         // get pilot's desired yaw rate
-        target_yaw_rate = get_pilot_desired_yaw_rate(channel_yaw->get_control_in());
+        target_yaw_rate_cdps = get_pilot_desired_yaw_rate(channel_yaw->get_control_in());
```

Comment:
target_yaw_rate is already in centi degree https://github.com/ArduPilot/ardupilot/blob/bcea3ac8d4e224661acdcd80c9318b77df30d342/ArduCopter/Attitude.cpp#L44 I don't think it is necessary to rename it but this decision is up to Randy